### PR TITLE
Feature/38670 bulk load test data

### PR DESCRIPTION
**Pull request by GuyHarwood**
scripts to bulk insert schools and pupils.

- [ ] refactor manual.sh to call ci-up.sh & ci-down.sh in sequence

### CODE REVIEW

File: deploy/load-test/build-db.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+SQL_SERVER=$1
+SQL_DATABASE=$2
+DB_SCALE=$3
+
+SQL_AZURE_FQDN="database.windows.net"
+
+### Run Migrations
+echo "running database migrations..."
+cd ../../admin
+yarn install
```
Comment (jon-shipley) [inline deploy/load-test/build-db.sh]:
```suggestion
yarn install --frozen-lockfile
```

File: deploy/sql/dummy-data/school.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    console.log('connected')
+    console.log(`inserting ${schoolCount} schools...`)
+    const request = new sql.Request(pool)
+    request.bulk(table, (err, result) => {
+      if (err) {
+        console.error(err.message)
+        process.exit(-1)
+      }
+      console.log('all done')
+      process.exit(0)
```
Comment (jon-shipley) [inline deploy/sql/dummy-data/school.js]: Would be better not to have to call this.  Does it not exit properly if you call drain()?

File: deploy/sql/package.json [hunk 3]
```diff
@@ -7,10 +7,13 @@
   "author": "Guy Harwood",
   "license": "GPL-3.0",
   "scripts": {
-    "db:up": "node ./sql/migrate.js",
-    "db:data": "node ./sql/seed.js",
-    "new:migration": "node ./sql/create-migration.js",
-    "new:seed": "node ./sql/create-seed.js"
+    "db:up": "node ./schema(WIP)/migrate.js",
```
Comment (jon-shipley) [inline deploy/sql/package.json]: I would prefer not to have special characters - the curly braces - in the directory name.

File: deploy/load-test/build-db.sh (hunk 1 above)
Comment (GuyHarwood) [reply deploy/load-test/build-db.sh]: Added to new script `do.sh`

File: deploy/sql/package.json (hunk 3 above)
Comment (GuyHarwood) [reply deploy/sql/package.json]: renamed

File: deploy/sql/dummy-data/school.js (hunk 2 above)
Comment (GuyHarwood) [reply deploy/sql/dummy-data/school.js]: done

**Pull request merged**